moviesurfer/cli/main.py

```diff
@@ -1,2 +1,16 @@
-def main():
-    print("Hello from CLI, functionality is to be added")
+import click
+
+from moviesurfer.data.make_dataset import make_dataset
+
+
+@click.command()
+@click.argument("input_filepath", type=click.Path(exists=True))
+@click.argument("output_filepath", type=click.Path())
+def main(input_filepath, output_filepath):
+    """Runs data processing scripts to turn raw data from (../raw) into
+    cleaned data ready to be analyzed (saved in ../processed).
+    """
+    make_dataset(input_filepath, output_filepath)
```

Comment:
for downloading data we have a nice function implemented
https://github.com/atsanda/movie-surfer/blob/main/moviesurfer/data/download_data.py#L12
feel free to use it